tests/unit/mixins/Registry.ts

```diff
@@ -57,7 +57,7 @@ registerSuite({
 			class IntegrationTest extends TestWithRegistry {
 				render() {
 					return v('div', [
-						w('test', { id: `${Math.random()}` })
+						w<any>('test', { id: `${Math.random()}` })
```

Comment:
When using the registry now, are we going to have to supply some sort of generic argument?